channels/rdp2tcp/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# FreeRDP: A Remote Desktop Protocol Implementation
+# FreeRDP cmake build script
+#
+# Copyright 2012 Marc-Andre Moreau <marcandre.moreau@gmail.com>
```

Comment:
sure about that copyright?

Reply:
I know that the code has long history and believe that this is one of first authors